src/modules/settings/pages/about.jsx

```diff
@@ -0,0 +1,141 @@
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
+function getDataURLFromUpload(input, callback) {
+  const reader = new FileReader();
+  reader.onload = ({target}) => callback(target.result);
+  const file = input.files[0];
+  if (!file) {
+    callback(null);
+    return;
+  }
+  reader.readAsText(file);
+}
+
+function importFile(target) {
+  getDataURLFromUpload(target, (data) => {
+    if (!isJSON(data)) return;
+
+    const settingsToImport = JSON.parse(data);
+    Object.keys(settingsToImport).forEach((s) => settings.set(s.split('bttv_')[1], settingsToImport[s]));
+
+    setTimeout(() => window.location.reload(), 1000);
+  });
+}
+
+function About() {
+  return (
+    <PanelGroup>
+      <Panel>
+        <Header heading="BetterTTV" description={`Version ${debug.version}`} />
+        <div style={{marginTop: 15}}>
+          Think this addon is awesome? Drop a Review on the{' '}
+          <a href="https://chrome.google.com/webstore/detail/betterttv/ajopnjidmegmdimjlfnijceegpefgped">
+            Chrome Webstore
+          </a>{' '}
+          or <a href="https://addons.mozilla.org/firefox/addon/betterttv/">Firefox Add-ons</a> site or maybe even
+          Subscribe to <a href="https://betterttv.com/dashboard/pro">BetterTTV Pro</a>!
+        </div>
+      </Panel>
+      <Panel>
+        <div className="socials">
+          <ul>
+            <li>
+              <p>EXPLORE</p>
+            </li>
+            <li>
+              <a href="https://betterttv.com/">Home</a>
+            </li>
+            <li>
+              <a href="https://github.com/night/betterttv/issues">Report Bugs</a>
+            </li>
+            <li>
+              <a href="https://github.com/night/betterttv/issues">Submit Ideas</a>
+            </li>
+            <li>
+              <a href="https://nightdev.com/">Other Products</a>
+            </li>
+          </ul>
+          <ul>
+            <li>
+              <p>COMMUNITY</p>
+            </li>
+            <li>
+              <a href="https://discord.gg/nightdev">Discord</a>
+            </li>
+            <li>
+              <a href="https://twitter.com/betterttv">Twitter</a>
+            </li>
+            <li>
+              <a href="https://community.nightdev.com/c/betterttv">Forums</a>
+            </li>
+            <li>
+              <a href="https://www.nightdev.com/contact">Email Us</a>
+            </li>
+          </ul>
+          <ul>
+            <li>
+              <p>LEGAL</p>
+            </li>
+            <li>
+              <a href="https://betterttv.com/terms">Terms of Service</a>
+            </li>
+            <li>
+              <a href="https://betterttv.com/privacy">Privacy Policy</a>
+            </li>
+          </ul>
+        </div>
+      </Panel>
+      <Panel header="Settings">
+        <div className="bttv-settings-backup">
+          <IconButton style={{width: '100%'}} appearance="primary" onClick={backupFile} icon={<Icon icon={download} />}>
```

Comment:
lets move inline styles to css files. this is the last comment i'll make. please search for `style=` and find all of them